features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin_new/source/src/app/components/Base/RouteMenuMapping.jsx

```diff
@@ -197,7 +197,7 @@ const RouteMenuMapping = (intl) => [
             {
                 id: intl.formatMessage({
                     id: 'Base.RouteMenuMapping.blacklisted.items',
-                    defaultMessage: 'Blacklisted Items',
+                    defaultMessage: 'Blacklist Rate Limiting Policies',
```

Comment:
I think rate limiting is not needed here since its blacklist policy.

Reply:
Fixed